### PR TITLE
Reworked the README

### PR DESCRIPTION
**Pull request by lyrixx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,8 +14,9 @@ composer require --dev jolicode/asynit
 
 #### Basic usage
 
-Asynit will read PHP's classes and try to mimic the API of PHPUnit, so you need to a create a test class in some directory,
-which will extends the `TestCase` class of Asynit:
+Asynit will read PHP's classes and try to mimic the PHPUnit API. Sso you need
```
Comment (kelunik) [inline README.md]: `Sso` → `So`, but I'd just remove the `So` entirely and write `You need …`

File: README.md [hunk 2]
```diff
@@ -14,8 +14,9 @@ composer require --dev jolicode/asynit
...
+to a create a test class in some directory, which will extends the `TestCase`
```
Comment (kelunik) [inline README.md]: `extends` → `extend`

**Comment by lyrixx** [comment]
Thanks @kelunik 

I have added a new commit about the tests organizations.

File: README.md [hunk 3]
```diff
@@ -29,34 +30,50 @@ Then you can add some tests that will use the API of the TestCase class:
...
+        $this->assertSame(200, $response->getStatusCode());
+        // or
+        $this->assertStatusCode(200, $response);
     }
 }
 ```
 
-Here we tell the test to do a GET request on `http://my-site-web` then we get the response by using the `yield` operator.
-This operator must but understand like an `await` in other language (C# / Javascript) which is feasible by using the amp framework.
+Note: All test methods should be prefixed by the `text` keyword. All others
```
Comment (kelunik) [inline README.md]: `test` :stuck_out_tongue_winking_eye:

File: README.md [hunk 4]
```diff
@@ -14,8 +14,9 @@ composer require --dev jolicode/asynit
 
 #### Basic usage
 
-Asynit will read PHP's classes and try to mimic the API of PHPUnit, so you need to a create a test class in some directory,
-which will extends the `TestCase` class of Asynit:
+Asynit will read PHP's classes and try to mimic the PHPUnit API. You need to a
```
Comment (pyrech) [inline README.md]: extra `a` at the end

File: README.md [hunk 5]
```diff
@@ -29,34 +30,50 @@ Then you can add some tests that will use the API of the TestCase class:
...
 ```
 
-Here we tell the test to do a GET request on `http://my-site-web` then we get the response by using the `yield` operator.
-This operator must but understand like an `await` in other language (C# / Javascript) which is feasible by using the amp framework.
+Note: All test methods should be prefixed by the `text` keyword. All others
+methods will not be executed automatically.
 
-Some assertions are given by this lib, but you can use your own as long as it's throw an exception on failure.
+Here we perform a `GET` request on `http://my-site-web` then we get the
+`$response` by using the `yield` operator. This operator must but understand
```
Comment (pyrech) [inline README.md]: Is `must but` correct?

Comment (lyrixx) [reply README.md]: `must be`, good catch.

**Pull request merged**